### DISCUSSION TITLE
There was a problem migrating the server

### DISCUSSION THREAD

**Discussion by tesfduwen** [NamelessMC/Nameless#3148] [Q&A] [answered]
I saved the database and website files locally before reinstalling the system on the server. After the server system was reinstalled, I put the data back. As a result, the website could not be accessed normally.
![屏幕截图 2022-12-29 180812.png](https://s2.loli.net/2022/12/29/NvFTpAose91XQKM.png)

**Comment by samerton** [comment, marked as answer]
Hi, can you please check the file `cache/logs/fatal-log.log` for further information

**Reply by tesfduwen** [reply]
Thank you, it has been solved. It's a problem with the database connection. I didn't know which folder the log was in before.
